packages/fx-core/tests/plugins/resource/bot/unit/expectedBicepFiles/botProvision.result.v2.bicep

```diff
@@ -1,5 +1,7 @@
+@secure()
```

Comment:
add it to provisionParameters instead of userAssignedIdnetityId?

Reply:
sure